### PR TITLE
Contentful: custom content model

### PR DESCRIPTION
**Pull request by AlbertGom**
## Description
Create `custom` content

## Context
There was no custom content model where you can add custom fields without having to be defined in the plugin.

## Approach taken / Explain the design
 Created a new component that has id,name and fields. Fields field is a record that has the custom fields defined in contentful.

## Testing

The pull request...

- has unit tests

### CODE REVIEW

**Review by Iru89 — APPROVED** [review]

**Review by asastre — APPROVED** [review]

**Comment by asastre** [comment]
Although I have approved the PR @AlbertGom take a look at the tests, they're failing!

**Comment by asastre** [comment]
The changes done before the last commit allowed to fetch custom content from Contentful only if the content model was named 'custom'.
With the changes done in the [last commit ](https://github.com/hubtype/botonic/pull/1900/commits/3b9d07d61c3a69a844eca54cb7203b6da6fbb0ff) allow fetching all type of content models different than the known ones (text, image, button, etc.).
@AlbertGom can you review this last commit?

**Review by AinaVendrell — APPROVED** [review]
🚀

File: packages/botonic-plugin-contentful/package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "scripts": {
     "build": "rm -rf lib && ../../node_modules/.bin/tsc",
+    "build:watch": "../../node_modules/.bin/tsc --watch",
```
Comment (vanbasten17) [inline packages/botonic-plugin-contentful/package.json:4]: in order to not duplicate the same command for this, you can do:
`"build:watch": "npm run build -- --watch"` to pass variables through a npm script

**Review by vanbasten17 — APPROVED** [review]

File: packages/botonic-plugin-contentful/package.json (hunk 1 above)
Comment (asastre) [reply packages/botonic-plugin-contentful/package.json:4]: I thought about it too, but in this case, there's no need to remove the `lib` folder when executing the build with the `--watch` flag, that's why I was "duplicating" the command.

Comment (vanbasten17) [reply packages/botonic-plugin-contentful/package.json:4]: as the watch process will be running in long term, I think that lib won't be removed every time (I haven't tested it, but I think it behaves like this). Not an important thing. Up to you!

**Pull request merged**